cddl/cbor-tags.txt

```diff
@@ -1,6 +1,8 @@
 tagged-corim-map = #6.501(corim-map)
+tagged-unsigned-corim = #6.501(unsigned-corim)
```

Comment:
1. and 2. are now super confusing to me!
 line 1 can be deleted presumably as no one uses tagged-corim-map ..?